### ISSUE TITLE
Support for LoRA finetuning.

### ISSUE THREAD

**Issue by ZenithWisp** [OpenGVLab/VideoChat-Flash#56]
Could you please confirm whether LoRA fine-tuning will be supported for the InternVideo model in the future? If yes, when can we expect it?

**Comment by leexinhao** [comment]
Try to modify some code here: https://github.com/OpenGVLab/VideoChat-Flash/blob/4ffe190100aba04e447780d30d42a101f1c49299/xtuner-train_internvideo2_5/unify_internvl2_train_r16.py#L614

Wrap model with LoRA, I think it's not hard, but I have no extra time and machine to verify it.

**Comment by Zhengxulion** [comment]
> Try to modify some code here:
> 
> [VideoChat-Flash/xtuner-train_internvideo2_5/unify_internvl2_train_r16.py](https://github.com/OpenGVLab/VideoChat-Flash/blob/4ffe190100aba04e447780d30d42a101f1c49299/xtuner-train_internvideo2_5/unify_internvl2_train_r16.py#L614)
> 
> Line 614 in [4ffe190](/OpenGVLab/VideoChat-Flash/commit/4ffe190100aba04e447780d30d42a101f1c49299)
> 
>  def build_llava_model(args, dtype=torch.float32, device='cpu'): 
> Wrap model with LoRA, I think it's not hard, but I have no extra time and machine to verify it.

Hi, thanks for the great work!
After packaging the visual module of the model with Lora, I encountered an issue during fine-tuning. 
`UserWarning: None of the inputs have requires_grad=True. Gradients will be None.  `
I want to know how to solve it. Looking forward to your reply.
